src/components/dashboard/EntityStatOverview/StatOverview.tsx

```diff
@@ -0,0 +1,155 @@
+import { Stack, Typography } from '@mui/material';
+import { PostgrestError } from '@supabase/postgrest-js';
+import { authenticatedRoutes } from 'app/routes';
+import LinkWrapper from 'components/shared/LinkWrapper';
+import {
+    semiTransparentBackground_blue,
+    semiTransparentBackground_purple,
+    semiTransparentBackground_teal,
+} from 'context/Theme';
+import { CloudDownload, CloudUpload, DatabaseScript } from 'iconoir-react';
+import { ReactElement } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { Entity } from 'types';
+import ActiveEntityCount from './ActiveEntityCount';
+import Statistic from './Statistic';
+
+interface Props {
+    entityType: Entity;
+    monthlyUsage?: number;
+    monthlyUsageError?: PostgrestError;
+    monthlyUsageLoading?: boolean;
+}
+
+const getEntityPageURLPath = (entityType: string): string => {
+    if (entityType === 'collection') {
+        return authenticatedRoutes.collections.fullPath;
+    }
+
+    if (entityType === 'capture') {
+        return authenticatedRoutes.captures.fullPath;
+    }
+
+    return authenticatedRoutes.materializations.fullPath;
+};
+
+const getTitleId = (entityType: string): string => {
+    if (entityType === 'collection') {
+        return 'terms.collections';
+    }
+
+    if (entityType === 'capture') {
+        return 'terms.sources';
+    }
+
+    return 'terms.destinations';
+};
+
+const getBackgroundColor = (
+    entityType: string
+): { dark: string; light: string } => {
+    if (entityType === 'collection') {
+        return semiTransparentBackground_blue;
+    }
+
+    if (entityType === 'capture') {
+        return semiTransparentBackground_teal;
+    }
+
+    return semiTransparentBackground_purple;
+};
+
+const getEntityIcon = (entityType: string): ReactElement => {
+    const iconSize = 12;
+
+    if (entityType === 'collection') {
+        return <DatabaseScript fontSize={iconSize} />;
+    }
+
+    if (entityType === 'capture') {
+        return <CloudUpload fontSize={iconSize} />;
+    }
+
+    return <CloudDownload fontSize={iconSize} />;
+};
```

Comment:
I think I figured out why this threw me off initially. These should really just return the actual component classes.

Then down below we use them as components

```
<Icon fontSize={iconSize} />
```